### PR TITLE
4451 add rd chart 3

### PR DESCRIPTION
**Pull request by brett-varney**
https://federal-spending-transparency.atlassian.net/browse/DA-4451
https://federal-spending-transparency.atlassian.net/browse/DA-6196

add chart & legend to R&D section 3

### CODE REVIEW

**Comment by anjenkin** [comment]
@brett-varney Looks good for the most part.  A couple things - 

1) The legend looks misaligned.  It's centered to the svg but it should be centered to the x-axis.  Should be able to add a margin left to the legend to accommodate for the text in the y-axis that's throwing the centering off.

2) The text font isn't Source Sans Pro in the svg

Screen Shot 2020-04-12 at 7 53 45 PM

Screen Shot 2020-04-12 at 7 51 15 PM

File: src/pages/rd-in-contracting/rd-in-contracting.module.scss [hunk 1]
```diff
@@ -1,7 +1,56 @@
...
+@media (min-width: $tablet) {
+  .legend-tile > div {
+    margin: 0 auto;
+    display: table;
+  }
+}
+
+@media (max-width: $tablet - 1) {
+  .legend {
+    padding-left: 10% !important;
```
Comment (anjenkin) [inline src/pages/rd-in-contracting/rd-in-contracting.module.scss:54]: is there anyway to get around using the important here?

**Comment by anjenkin** [comment]
@brett-varney Also just wondering why we're not following the pattern we've been using with page-sections?  If we don't use page-sections here we could end up with a lot of conflicts as three of us are working on the same page.  Also, it's a little confusing because it's different than all the other pages.

**Comment by brett-varney** [comment]
>     2. The text font isn't Source Sans Pro in the svg
All the text nodes in the SVG are either "SourceSansPro-Bold, Source Sans Pro" or "SourceSansPro-Light, Source Sans Pro"; what's changing that? Gatsby?

File: src/pages/rd-in-contracting/rd-in-contracting.module.scss (hunk 1 above)
Comment (brett-varney) [reply src/pages/rd-in-contracting/rd-in-contracting.module.scss:54]: No, why?

**Comment by brett-varney** [comment]
>     1. The legend looks misaligned.  It's centered to the svg but it should be centered to the x-axis.  Should be able to add a margin left to the legend to accommodate for the text in the y-axis that's throwing the centering off.

Done, as best as I can.

**Comment by brett-varney** [comment]
> 
> 
> @brett-varney Also just wondering why we're not following the pattern we've been using with page-sections? If we don't use page-sections here we could end up with a lot of conflicts as three of us are working on the same page. Also, it's a little confusing because it's different than all the other pages.

That'll be much easier when I move it to the story template. This is just to show the chart and legend.

File: src/pages/rd-in-contracting/rd-in-contracting.module.scss (hunk 1 above)
Comment (anjenkin) [reply src/pages/rd-in-contracting/rd-in-contracting.module.scss:58]: Generally !important shouldn't be used unless it is absolutely necessary after all other options have been exhausted

**Comment by anjenkin** [comment]
@brett-varney The legend looks good.  The text in the svg is still not Source Sans Pro.  Without looking at this more in depth, I would think it has to do with how the svg was created.   I can merge this in once the merge conflicts are resolved.  That would be great if you could log a follow up ticket to fix the font in the svg.

File: src/pages/rd-in-contracting/rd-in-contracting.module.scss (hunk 1 above)
Comment (brett-varney) [reply src/pages/rd-in-contracting/rd-in-contracting.module.scss:58]: Agreed, I just can't always find a way around MUI's specificity. I can give it another shot...

Comment (brett-varney) [reply src/pages/rd-in-contracting/rd-in-contracting.module.scss:58]: Problem here (and other places) is MUI styles are coming after ours, so MUI wins with equal specificity. MUI also applies some styles directly to page elements without classes, so overrides any class-based styles. So I used the simplest and most-specific override I could find.

I suppose a less-hacky thing to do is create a MUI style override and use it here.

**Comment by brett-varney** [comment]
@anjenkin conflict resolved, explanation of !important. You can merge, or we can do the MUI style override differently next sprint.

**Pull request merged**